projects/novo-elements/src/elements/tabbed-group-picker/TabbedGroupPicker.ts

```diff
@@ -225,7 +235,18 @@ export class NovoTabbedGroupPickerElement implements OnDestroy, OnInit {
     }
   }
 
-  onItemToggled(item: { selected?: boolean; children?: Array<{ selected?: boolean; children?: Array<{ selected?: boolean }> }> }) {
+  activateItem(item: any): void {
+    if (item.disabled) {
```

Comment:
I need to take this out. I was initially introducing a disabled state, but we then decided to slow down on any directly added features.

Reply:
Code looks good. Once you are done with this comment, I can approve